testpmd/hooks/user-tests.yml

```diff
@@ -3,3 +3,10 @@
   include_role:
     name: example-cnf-validate
   when: run_migration_test|default(true)|bool
+
+- name: Get logs from pods
+  include_role:
+    name: get-logs-from-namespace
```

Comment:
Where can i find this role? Although, I would have named it as `collect-namespace-logs`

Reply:
https://softwarefactory-project.io/r/#/c/20121/